### PR TITLE
title1

### PR DESCRIPTION
**Pull request by testing-in-chrome**
body1

### CODE REVIEW

**Comment by testing-in-chrome** [comment]
lgtm

**Comment by testing-in-chrome** [comment]
+1

File: ghcl.py [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
   print(gh_repo.commits_url)
 
-  pr = gh_repo.create_pull("title1", "body1", "master",
-                           str(repo.active_branch))
-  print pr.url()
\ No newline at end of file
+  print(gh_repo.pulls_url)
+  pr = gh_repo.create_pull(title="title1", body="body1", base="master",
+                           head=branch)
+  print pr.url
```
Comment (testing-in-chrome) [inline ghcl.py:25]: Good fix.

File: ghcl.py [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
   repo = Repo(os.getcwd())
   assert not repo.bare
-  g = Github()
+  username = raw_input("Enter Github username: ")
+  password = getpass.unix_getpass("Password: ")
+  g = Github(username, password)
   urls = []
   for url in repo.remote().urls:
     urls.append(url)
   gh_repo = g.get_repo("testing-in-chrome/test_buck")
```
Comment (testing-in-chrome) [inline ghcl.py:15]: Read this value from a settings file in the future.

**Review by dev-in-chrome — APPROVED** [review]

**Pull request merged**